CHANGE_LOG.md

```diff
@@ -2,9 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
-# Chat Widget
+# Chat-Widget
 
 ## [Unreleased]
+### Added 
```

Comment:
I don't think this changelog entry is necessary, since this is not a product change. (just like we don't update changelog for documentation PRs

Reply:
yeah, I was curious about this one as well... Ill remove it